composer.json

```diff
@@ -23,21 +23,21 @@
         "premium-wp-theme/genesis": "~2.1",
         "wp-sync-db/wp-sync-db": "dev-master",
         "wp-sync-db/wp-sync-db-media-files": "dev-master",
-        "wpackagist-plugin/akismet": "~3.0",
+        "wpackagist-plugin/akismet": "~3.1",
         "wpackagist-plugin/autoptimize": "~2.0",
         "wpackagist-plugin/black-studio-tinymce-widget": "~2.2",
         "wpackagist-plugin/disable-emojis": "~1.5",
         "wpackagist-plugin/force-regenerate-thumbnails": "~2.0",
-        "wpackagist-plugin/google-analytics-for-wordpress": "~5.3",
-        "wpackagist-plugin/wordpress-seo": "~3.0",
+        "wpackagist-plugin/google-analytics-for-wordpress": "~5.5",
+        "wpackagist-plugin/wordpress-seo": "~3.4",
         "composer-plugin-api": "^1.1"
     },
     "require-dev": {
```

Comment:
@xipasduarte Sorry, I didn't saw them above. But they should be a `require-dev` dependency.